packages/api-client/src/account/AccountAPI.ts

```diff
@@ -130,6 +130,34 @@ export class AccountAPI {
     await this.client.sendJSON(config);
   }
 
+  /**
+   * Verify phone sms code
+   * @see https://staging-nginz-https.zinfra.io/swagger-ui/#!/activate
+   */
+  public async postVerifyPhoneCode(phone: string, code: string): Promise<boolean> {
+    const config: AxiosRequestConfig = {
+      method: 'post',
+      params: {
+        code,
+        dryrun: true,
+        phone,
+      },
+      url: AccountAPI.URL.ACTIVATE,
+    };
+    try {
+      const response = await this.client.sendJSON(config);
+      if (response.status === 200) {
+        return true;
+      }
+      if (response.status === 404) {
+        return false;
+      }
+      return false;
+    } catch (error) {
+      throw error;
```

Comment:
try/catch is not needed if you are not doing anything with it.

Reply:
done in https://github.com/wireapp/wire-web-packages/pull/3050/commits/ea1aa0c2f970faf53850df0392e6a1ea566e6b42